lib/site_prism/element_container.rb

```diff
@@ -71,6 +71,20 @@ def raise_if_block(obj, name, has_block)
       raise SitePrism::UnsupportedBlock, "#{obj.class}##{name}"
     end
 
+    def raise_wait_for_if_failed(obj, name, timeout, failed)
+      return unless SitePrism.raise_on_wait_fors && failed
+
+      raise SitePrism::TimeOutWaitingForExistenceError, \
+            "Timed out after #{timeout}s waiting for #{obj.class}##{name}"
```

Comment:
These messages are very descriptive!  
But I feel like they're perhaps unnecessary and just one more thing to maintain (part of the reason why it got turned into its own method).  
The stacktrace already gives you the exact test code line the error occurs on and because of the design of SitePrism - it's already descriptive: 
`@test_site.home.wait_for_some_slow_element(5)` tells me everything in this message.

Reply:
Thanks, I mainly included this to match the behavior of the rest of the gem (timeout message and unsupported block message) and because I think it can be helpful for productivity in some cases, for example when the errors in Jenkins with a JUnit formatter, the Test Results page will show the message of the error and someone doesn't need to read the stack trace, but I agree the stacktrace contains the class and element information.

Do you think I should just include the timeout duration information?